.github/workflows/build-pygtk.yaml

```diff
@@ -61,9 +67,8 @@ jobs:
         uses: actions/upload-artifact@v4
         with:
           name: ${{ env.GTK_BUILD_NAME }}
-          path: vcpkg_installed
+          path: ${{ env.GTK_BUILD_NAME }}.zip
```

Comment:
@mkhon 
This action always archives the path, so it makes a zip within a zip. Is that intended?

```
$ unzip -l gtk3.24-x86-windows.zip 
Archive:  gtk3.24-x86-windows.zip
  Length      Date    Time    Name
---------  ---------- -----   ----
 84371435  2025-03-27 10:24   gtk3.24-x86-windows.zip
---------                     -------
 84371435                     1 file
```

This does not affect the release, though,

Reply:
yes, this is intentional. there is no other way